test/verify/check-machines-dbus

```diff
@@ -282,7 +328,7 @@ class TestMachinesDBus(machineslib.TestMachines):
 
         # Check overview network properties are present
         b.wait_in_text("#network-test_network2-{0}-persistent".format(connectionName), "no")
-        b.wait_in_text("#network-test_network2-{0}-autostart".format(connectionName), "no")
+        b.wait_not_present("#network-test_network2-{0}-autostart-checkbox:checked".format(connectionName))
```

Comment:
wait_not_present can be try if the whole element is missing.
We want the opposite logic here. Wait present and with not checked attribute. Something like this ':not(:checked)' should work I think